src/utils.js

```diff
@@ -6,6 +6,14 @@ import cl100k_base from 'js-tiktoken/ranks/cl100k_base';
 const OUTPUT_DIR = './.cache';
 let encoder;
 
+export async function loadBundles(url, date, domainkey, ckpt) {
```

Comment:
```suggestion
export async function loadBundles(url, date, domainkey) {
```
nitpick, `ckpt` isn't used